README.md

```diff
@@ -1,4 +1,87 @@
-# Hello_Microverse-Pro
+# Hello-Microverse-Project
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+Hello-Microverse-Project
+
+> Describe your project in 1 or 2 sentences.
```

Comment:
[OPTIONAL]
- [ ] Kindly delete any instructions since they were only for your guidance, leaving them makes your `README` looks unprofessional.